Unable to exec py27 tox suite on macOS/Homebrew

```
$ tox -e py27 -- -k test_creator
[...]

====================================================================================== ERRORS =======================================================================================
____________________________________________________ ERROR collecting tests/unit/discovery/py_info/test_py_info_exe_based_of.py _____________________________________________________
tests/unit/discovery/py_info/test_py_info_exe_based_of.py:20: in <module>
    BASE = {str(Path(CURRENT.executable).parent.relative_to(Path(CURRENT.prefix))), "."}
/usr/local/lib/python2.7/site-packages/pathlib2/__init__.py:1132: in relative_to
    .format(str(self), str(formatted)))
E   ValueError: '/usr/local/opt/python@2/bin' does not start with '/usr/local/Cellar/python@2/2.7.17_1/Frameworks/Python.framework/Versions/2.7'
-------------------------------------------------------- generated xml file: /Users/jd/Source/virtualenv/.tox/junit.py27.xml --------------------------------------------------------
============================================================================== short test summary info ==============================================================================
FAILED tests/unit/discovery/py_info/test_py_info_exe_based_of.py - ValueError: '/usr/local/opt/python@2/bin' does not start with '/usr/local/Cellar/python@2/2.7.17_1/Frameworks/P...
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
====================================================================== 134 deselected, 1 error in 0.45 seconds ======================================================================
Coverage.py warning: Module /Users/jd/Source/virtualenv/.tox/py27/lib/python2.7/site-packages/virtualenv was never imported. (module-not-imported)
Coverage.py warning: No data was collected. (no-data-collected)
ERROR: InvocationError for command /Users/jd/Source/virtualenv/.tox/py27/bin/python -m coverage run -m pytest --junitxml .tox/junit.py27.xml tests -k test_creator (exited with code 2)
______________________________________________________________________________________ summary ______________________________________________________________________________________
ERROR:   py27: commands failed
```

I think the problem is that the path is a symlink:

```
$ ls -dl /usr/local/opt/python@2
lrwxr-xr-x  1 jd  admin  27 Dec 29 19:09 /usr/local/opt/python@2 -> ../Cellar/python@2/2.7.17_1
```

However, I realized it breaks for Python 3.7:

```
E   ValueError: '/usr/local/Cellar/python/3.7.6_1/Frameworks/Python.framework/Versions/3.7/bin' does not start with '/Users/jd/Source/virtualenv/.tox/py37'
```

--- a/tests/unit/discovery/py_info/test_py_info_exe_based_of.py
+++ b/tests/unit/discovery/py_info/test_py_info_exe_based_of.py
@@ -7,7 +7,6 @@ import pytest
 
 from virtualenv.discovery.py_info import EXTENSIONS, PythonInfo
 from virtualenv.info import IS_WIN, fs_is_case_sensitive, fs_supports_symlink
-from virtualenv.util.path import Path
 
 CURRENT = PythonInfo.current()
 
@@ -17,7 +16,7 @@ def test_discover_empty_folder(tmp_path, monkeypatch, session_app_data):
         CURRENT.discover_exe(session_app_data, prefix=str(tmp_path))
 
 
-BASE = {str(Path(CURRENT.executable).parent.relative_to(Path(CURRENT.prefix))), "."}
+BASE = {CURRENT.distutils_install["scripts"], "."}
 
 
 @pytest.mark.skipif(not fs_supports_symlink(), reason="symlink is not supported")

